src/tiled/mapeditor.cpp

```diff
@@ -479,7 +480,10 @@ void MapEditor::performStandardAction(StandardAction action)
         paste(ClipboardManager::PasteInPlace);
         break;
     case DeleteAction:
-        MapDocumentActionHandler::instance()->delete_();
+        if (mEditPolygonTool->countSelectedHandles())
```

Comment:
I personally do not like these implicit > 0 checks on integer values in this context. I usually prefer to write `> 0` here. But not sure what @bjorn. Thinks here...